.travis.yml

```diff
@@ -54,10 +54,15 @@ matrix:
       compiler: gcc
       env: VALGRIND=ON CHECK_EXPORTS=ON
       addons: *linux64
+    - name: cmake-2.8.12
+      os: linux
+      compiler: gcc
+      env: ALTERNATIVE_CMAKE=2.8.12-Linux-i386
+      addons: *linux64
     - name: 32-bits
       os: linux
       compiler: gcc
-      env: ARCH=x86 CMAKE_EXTRA=-DHAVE_LIBM=/lib32/libm.so.6
+      env: ARCH=x86 CMAKE_EXTRA=-DHAVE_LIBM=/lib32/libm.so.6 ALTERNATIVE_CMAKE=2.8.12-Linux-i386
```

Comment:
Is there a reason that the 32-bit build is only done with the old CMake?

Reply:
No, just tried to use cmake-2.8 with an existing build. Since it was failing, I moved it to its own build, but now both are failing ...

Reply:
OK, I got confused here. The 32-bit builder should not have used cmake 2.8.12